lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Surface.rb

```diff
@@ -8,7 +8,7 @@ class ASHRAE901PRM < Standard
   def surface_adjust_fenestration_in_a_surface(surface, reduction, model)
     # do nothing for cases when reduction == 1.0
     if reduction < 1.0
-      surface.subSurfaces.sort.each do |ss|
+      surface.subSurfaces.each do |ss|
```

Comment:
There are some pieces of code which give different results depending on whether .sort is called or not. If the object is being removed, you probably don't need it, but it may be worth including the sort to be safe.

Reply:
Got it. Thanks for pointing it out. I will check and add them back.